api/users/services/addUser.js

```diff
@@ -0,0 +1,15 @@
+const { hashPassword } = require("../../../services");
+const { User } = require("../../../models");
+
+const addUser = async userObject => {
+  const hashedPassword = await hashPassword(userObject.password);
```

Comment:
Usually for user creation we use password and repeatPassword fields

Reply:
fixed